src/containers/Accounts/Login.js

```diff
@@ -108,34 +116,54 @@ class Login extends Component {
       defaultMessage: 'Password',
     });
 
+    const GoogleButton = withStyles((theme) => ({
```

Comment:
I wonder how these hooks worked for you but threw an error for me. I thought they weren't supposed to work in class-type components.

Reply:
Whatever -- now I can just copy this!